src/BUILD.gn

```diff
@@ -66,6 +66,8 @@ if (chip_build_tests) {
       "${chip_root}/src/protocols/interaction_model/tests",
       "${chip_root}/src/protocols/user_directed_commissioning/tests",
       "${chip_root}/src/transport/retransmit/tests",
+      "${chip_root}src/app/icd/server/tests",
```

Comment:
Should these tests appear in openiot cmake settings now that these start being visible in the main test list? like https://github.com/project-chip/connectedhomeip/blob/master/src/test_driver/openiotsdk/unit-tests/test_components_nl.txt and https://github.com/project-chip/connectedhomeip/blob/master/src/test_driver/openiotsdk/unit-tests/test_components.txt ?

Reply:
Thanks for the reminder! added the libs to the correct files